### PR TITLE
Update the staking & unstake event

### PR DESCRIPTION
**Pull request by 0xtaichi**
## Update

1. Add Staking and Unstakinig event into staking contract

### CODE REVIEW

File: contracts/Staking.sol [hunk 1]
```diff
@@ -561,6 +561,9 @@ contract OlympusStaking is Ownable {
     address public warmupContract;
     uint public warmupPeriod;
 
+    event Stake(uint _amount, address _recipient);
```
Comment (0xkowloon) [inline contracts/Staking.sol:564]:
```suggestion
    event Stake(address indexed recipient, uint amount);
```

File: contracts/Staking.sol [hunk 2]
```diff
@@ -561,6 +561,9 @@ contract OlympusStaking is Ownable {
...
+    event UnStake(uint _amount, bool _trigger);
```
Comment (0xkowloon) [inline contracts/Staking.sol:565]:
```suggestion
    event Unstake(address indexed staker, uint amount);
```

is the trigger argument really necessary?

File: contracts/Staking.sol (hunk 1 above)
Comment (0xtaichi) [reply contracts/Staking.sol:564]: I actually followed the original ohm setting
```
# callHandlers:
#   - function: stake(uint256,address)
#     handler: handleStake
#   - function: unstake(uint256,bool)
#     handler: handleUnstake
```

File: contracts/Staking.sol (hunk 2 above)
Comment (0xtaichi) [reply contracts/Staking.sol:565]: same as above, but trigger seems not necessary

File: contracts/Staking.sol (hunk 1 above)
Comment (0xtaichi) [reply contracts/Staking.sol:564]: And as far as I know, the eventHandler really for rinkeby testing purpose, we may not need those events for mainnet.

**Pull request merged**